skimage/graph/__init__.py

```diff
@@ -1,3 +1,7 @@
+"""
+Utilities for Image Segmentation, Object detection and Network Analysis.
+This module provides functions to work with graphs including finding the shortest path,closest centrality and merging.
```

Comment:
I like this bit but I think the wording and grammar can be tweaked to give more insight. How about

```suggestion
This module provides functions for graph-based analysis of images and
segmentations, e.g. path finding and cost evaluation, merging and cutting of
graphs, or finding the closest centrality.
```

The first part highlights what the graphs are used for and is a bit more specific. What do you think?